packages/data-context/src/actions/LocalSettingsActions.ts

```diff
@@ -14,19 +14,25 @@ export interface LocalSettingsApiShape {
 export class LocalSettingsActions {
   constructor (private ctx: DataContext) {}
 
-  setPreferences (stringifiedJson: string, type: 'global' | 'project') {
+  async setPreferences (stringifiedJson: string, type: 'global' | 'project') {
     const toJson = JSON.parse(stringifiedJson) as AllowedState
 
-    // update local data
-
-    _.merge(this.ctx.coreData.localSettings.preferences, toJson)
     if (type === 'global') {
+      // update local data on server
```

Comment:
Actually not too sure now that I look at it, what `refreshLocalSettings` does is slightly different since there's no "new" state that we want to put in that cache, it looks like we only use this function to populate it with whatever is on disk when Cypress is opened. A comment here might be confusing (though I'm open if you have a specific comment you think should be added).